Styles/nav.css

```diff
@@ -0,0 +1,37 @@
+.navigation-item{
+    padding:20px;
+    border: 0.5px solid rgba(128, 128, 128, 0.363);
+    margin: 20px;
+}
+.nav-item{
+    display:flex;  
+    align-items: center;           
+    background-color: rgb(209, 201, 201);
+
+}
+.left-nav{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width:700px;
```

Comment:
You could use % it helps in responsiveness

Reply:
Thanks bro for suggestion.